packages/model-instance-client/src/client.ts

```diff
@@ -84,4 +88,24 @@ export class DocumentClient extends StreamClient {
     const cid = await this.ceramic.postEventType(SignedEvent, event)
     return CommitID.fromStream(params.currentID.baseID, cid)
   }
+
+  /** Retrieve and return document state */
+  async getDocumentState(streamID: string): Promise<DocumentState> {
+    const streamState = await this.getStreamState(streamID)
+    const encodedData = streamState.data
+
+    const decodedData = decodeBase64urlToJSON(encodedData)
```

Comment:
Do you have tests that show this works? I would expect this to fail for two reasons.

1. The data string is prefixed with `u` and this decode method doesn't remove it
2. We cannot always assume that we get base64url encoded data, we need to use the mutliformats package to handle this.


The real problem is 2, and fixing 2 will fix 1.  This should be how to decode the multibase data.

```
import { bases } from 'multiformats/basics';

// Function to decode a multibase string into bytes
function decodeMultibase(multibaseString: string): Uint8Array {
  const baseDecoder = bases.decode(multibaseString);
  return baseDecoder.bytes;
}
```